modules/ref/pages/client-settings.adoc

```diff
@@ -531,6 +531,21 @@ TIP: Most client operations that accept an options block allow for overriding th
 +
 See the advanced section in the documentation on more specific information on retry strategies and failure management.
 
+[#unordered-executions]
+Name: *Unordered Execution*::
+Builder Method: `unorderedExecutionEnabled(boolean)`
++
+Default:  _true_
++
+System Property: `com.couchbase.unorderedExecutionEnabled`
++
+From Couchbase 7.0, Out-of-Order returns prevent operations from being blocked by long-running KV requests, potentially improving performance for durable writes and multi-document ACID transactions.
```

Comment:
It's all correct but maybe "prevent" & "blocked" are a touch negative?  A more positive way of writing this could be "From Couchbase 7.0, Out-of-Order execution allows the server to concurrently handle multiple requests on the same connection, potentially improving performance for durable writes and multi-document ACID transactions."?

Reply:
Sounds good  :+1: